backend/src/main/kotlin/app/ehrenamtskarte/backend/common/utils/Environment.kt

```diff
@@ -0,0 +1,8 @@
+package app.ehrenamtskarte.backend.common.utils
+
+// This helper class was created to enable mocking getenv in Tests
```

Comment:
alternatively, we can set env variables for tests in gradle configuration:

```
tasks.withType<Test> {
    environment("KOBLENZ_PEPPER", "123456789ABC")
}
```

Reply:
Thanks for sharing this idea, i tried different solutions and result of my research was that this mockable helper was the most robust solution.